Makefile

```diff
@@ -24,6 +24,7 @@ clean-build:
 	rm -fr *.egg-info
 	rm -fr .eggs
 	rm -fr .cache
+	rm -r kolibri/dist/exercise_perseus_renderer || true
```

Comment:
This error:

```
fatal: Not a git repository: kolibri/dist/exercise_perseus_renderer/static/mathjax/../../../../../.git/modules/perseus/modules/lib/mathjax
```

is being thrown by `git clean -X -d -f kolibri/dist` because for some reason there's a file called `.git` in the _kolibri/dist/exercise_perseus_renderer/static/mathjax/_ folder with the contents:

```
gitdir: ../../../../../.git/modules/perseus/modules/lib/mathjax
```

I don't know why this exists, but it seems maybe related to submodules?

----

In any case, I would recommend:

1. changing `rm -r kolibri/dist/exercise_perseus_renderer` to `rm -r kolibri/dist/` to remove all these files
2. adding `kolibri/dist` to _.gitignore_
3. manually generating `kolibri/dist/__init__.py` at the beginning of the build process.

That way we don't have any `dist` stuff checked into the repo, and everything gets cleaned up without needing special logic to handle plugins.